web/gds-user-ui/src/modules/dashboard/member/service/index.tsx

```diff
@@ -0,0 +1,13 @@
+import axiosInstance from 'utils/axios';
+import { DirectoryType } from '../memberType';
+
+const getVaspDirectory = (dir: DirectoryType) => {
+  return dir === 'mainnet' ? 'vaspdirectory.net' : 'trisatest.net';
```

Comment:
`vaspdirectory.net` or `vaspdirectory.dev` could be provided as the `registered_directory` for `mainnet`. Also,  `trisatest.net` or `trisatest.dev` could be the `registered_directory` for `testnet`. Could this be updated?

Reply:
Thanks. According to Ben, either is okay.